frontend/src/TopBar.js

```diff
@@ -0,0 +1,130 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {
+  AppBar,
+  Toolbar,
+  makeStyles,
+  Typography,
+  Button,
+  IconButton,
+  MenuItem,
+  Drawer,
+  Icon,
+  Divider
+} from '@material-ui/core';
+import {AccountCircle, Menu} from '@material-ui/icons'
+import {Route,BrowserRouter as Router} from 'react-router-dom';
+import {ThemeProvider,createMuiTheme} from '@material-ui/core/styles'
+import {green} from '@material-ui/core/colors'
+import {colors } from '@material-ui/core';
+import {Link} from 'react-router-dom';
+
+
+const useStyles = makeStyles(({
+  root: {},
+  toolbar: {
+    height: 64
+  }
+}));
+
+const theme = createMuiTheme({
+  palette:{
+    primary:{
+      main: green[500]
+    },
+    secondary: {
+      main: colors.lightBlue[500]
+    },
+    text: {
+      primary: colors.blueGrey[900],
+      secondary: colors.blueGrey[600]
+    }
+  },
+
+  typography :{
+    h1:{
+      fontFamily: "sans-serif",
+      fontSize:"1rem"
+    },
+    h3:{
+    fontFamily: "sans-serif",
+    fontSize:"1.3rem"
+    }
+  }
+
+  }
+)
+
+
+const TopBar = () => {
+  const [open,setOpen] = useState(false)
+  const handleDrawer=()=>{
+    setOpen(true)
+  }
+  const classes = useStyles();
+  return (
+    <ThemeProvider theme = {theme}> 
+    <Router>
```

Comment:
We've already defined a router elsewhere in the application so defining another here could cause problems.